pkg/services/postgresql/catalog.go

```diff
@@ -75,8 +101,24 @@ func createGPPlan(
 		defaultBackupRetention:  7,
 		tier: "GP",
 	}
+
+	updateSchema := planSchema{
+		allowedSSLEnforcement:  []string{enabledParamString, disabledParamString},
```

Comment:
These are largely the same as the provision metadata defs, but the omit the backup redundancy and hardware type.  This is true for all of them.